com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -197,6 +219,10 @@ internal static class ShaderPropertyId
         public static readonly int inverseCameraProjectionMatrix = Shader.PropertyToID("unity_CameraInvProjection");
         public static readonly int worldToCameraMatrix = Shader.PropertyToID("unity_WorldToCamera");
         public static readonly int cameraToWorldMatrix = Shader.PropertyToID("unity_CameraToWorld");
+
+        public static readonly int sourceTex = Shader.PropertyToID("_SourceTex");
+        public static readonly int scaleBias = Shader.PropertyToID("_ScaleBias");
```

Comment:
Suggestion: name this _ScaleBiasUV for better clarify.

Reply:
We can address this in the next PR.